docs/concepts/materializing-features.md

```diff
@@ -120,8 +120,8 @@ settings = MaterializationSettings("nycTaxiTable",
                                    backfill_time=backfill_time)
 ```
 
-This will generate features from `2020/05/10` to `2020/05/20` and the output will have 11 folders, from
-`abfss://feathrazuretest3fs@feathrazuretest3storage.dfs.core.windows.net/materialize_offline_test_data/df0/daily/2020/05/10` to `abfss://feathrazuretest3fs@feathrazuretest3storage.dfs.core.windows.net/materialize_offline_test_data/df0/daily/2020/05/20`. Note that currently Feathr only supports materializing data in daily step (i.e. even if you specify an hourly step, the generated features in offline store will still be presented in a daily hierarchy).
+This will generate features up from a starting date up to `2020/05/10` and `2020/05/20` correspondingly, and the output will have 11 folders, from
```

Comment:
"This will generate features up from a starting date up to `2020/05/10` and `2020/05/20` correspondingly". This sentence looks a little bit wired...

Reply:
yeah it is a bit confusing.. I've updated those.